lab_3_keywords_textrank/main.py

```diff
@@ -564,7 +657,8 @@ def get_top_keywords(self, n_keywords: int) -> tuple[int, ...]:
             tuple[int, ...]
                 top n most important tokens in the encoded text
         """
-        pass
+        sorted_key = sorted(self._scores)
```

Comment:
although dictionaries are in fact ordered in the later versions of python, it is better to establish secondary sorting order using `key` argument of the `sorted` fuction